app/views/catalog/show.html.erb

```diff
@@ -1,6 +1,11 @@
 <% @page_title = t('blacklight.search.show.title',
   document_title: truncate(strip_tags(document_presenter(@document).html_title).html_safe, length: 50), application_name: application_name).html_safe %>
-<% content_for(:head) { render_link_rel_alternates } %>
+
+<% content_for(:head) do %>
+  <meta name="turbo-visit-control" content="reload">
```

Comment:
This seems like a really big stick.  Is it possible to put `data-turbo="false"` on the affected forms instead?   I think this is more difficult to reason about as we don't have a clue as to where these ajax requests may be originating.

Reply:
changed.